randovania/games/samus_returns/logic_database/Surface - East.txt

```diff
@@ -818,14 +864,18 @@ Extra - asset_id: collision_camera_019
   * Pickup 2; Category? Minor
   * Extra - actor_name: LE_Item_003
   * Extra - actor_type: item_supermissiletank
+  > Dock to Landing Site (Top)
+      Space Jump
   > Dock to Landing Site (Bottom)
       Trivial
 
 > Dock to Landing Site (Top); Heals? False
   * Layers: default
   * Open Passage to Landing Site/Dock to Surface Crumble Block Challenge (Top)
   > Pickup (Super Missile Tank)
-      Trivial
+      Any of the following:
+          Phase Drift or Space Jump or Movement (Beginner)
+          Spider Boost (Intermediate) and Can Spider Boost
```

Comment:
Add comment for Movement. Something like hold right after falling off the ledge. Maybe add comment for where to Boost from (left side)